app/lib/actions/katello/repository/index_content.rb

```diff
@@ -2,21 +2,20 @@ module Actions
   module Katello
     module Repository
       class IndexContent < Actions::EntryAction
-        middleware.use Actions::Middleware::ExecuteIfContentsChanged
-
         input_format do
           param :id, Integer
-          param :dependency, Hash
-          param :contents_changed
```

Comment:
overall this looks good, but the main sync action, uses the contents_changed as a 'force' flag here: 

https://github.com/Katello/katello/blob/master/app/lib/actions/katello/repository/sync.rb#L50-L52

so it might be helpful to have an actual 'force' flag that can be passed in to skip the if timestamp check (that would be more clear too)

Reply:
That is the sort of thing that was nagging at my mind!